java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1171,6 +1171,39 @@ public void zscore(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zdiffstore(BaseClient client) {
+        String key1 = "{testKey:}" + UUID.randomUUID();
+        String key2 = "{testKey:}" + UUID.randomUUID();
+        String key3 = "{testKey:}" + UUID.randomUUID();
+        String key4 = "{testKey:}" + UUID.randomUUID();
+        String key5 = "{testKey:}" + UUID.randomUUID();
+
+        Map<String, Double> membersScores1 = Map.of("one", 1.0, "two", 2.0, "three", 3.0);
+        Map<String, Double> membersScores2 = Map.of("two", 2.0);
+        Map<String, Double> membersScores3 = Map.of("one", 0.5, "two", 2.0, "three", 3.0, "four", 4.0);
+
+        assertEquals(3, client.zadd(key1, membersScores1).get());
+        assertEquals(1, client.zadd(key2, membersScores2).get());
+        assertEquals(4, client.zadd(key3, membersScores3).get());
+
+        assertEquals(2, client.zdiffstore(key4, new String[] {key1, key2}).get());
+        assertEquals(1, client.zdiffstore(key4, new String[] {key3, key2, key1}).get());
+
+        assertEquals(0, client.zdiffstore(key4, new String[] {key1, key3}).get());
+        assertEquals(0, client.zdiffstore(key4, new String[] {key1, key1}).get());
+
```

Comment:
Add tests where
* source key is missing
* dest is one of the source keys
* with 1 and 0 source keys

Reply:
I believe most of these tests are not needed. When a key is missing they are treated as an empty set. Testing all of these scenarios is testing redis at this point and is confusing for someone actually reading the tests. I have added a test for a key that does not exist.